js/background.js

```diff
@@ -44,6 +49,14 @@ chrome.omnibox.onInputEntered.addListener(function(text) {
         sidURL='http://odo.corp.qualtrics.com/?b=ProductToolsSurveySearch&autoSurvey=' + text.substring(4, (text.length));
         chrome.tabs.create({url: sidURL });
       }
+      // else if (text.indexOf('sumos ')==0){
```

Comment:
Typically it's good practice to remove old code then comment it out. Is there a reason for keeping this code? If not, go ahead and remove it.